<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizedModelDialog</class>
 <widget class="QWidget" name="CustomizedModelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SetColor_Btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Render_Btn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Render</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_Btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cell Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CellType_comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
